components/ChipTextField/examples/ChipTextFieldExampleViewController.swift

```diff
@@ -0,0 +1,126 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+class ChipTextFieldExampleViewController: UIViewController {
+
+  let chipTextField = MDCChipTextField()
+  let chipInputController: MDCTextInputControllerOutlined
+  let textField = MDCTextField()
+  let inputController: MDCTextInputControllerOutlined
+  let resignButton = MDCButton()
+
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    chipInputController = MDCTextInputControllerOutlined(textInput: chipTextField)
+    chipInputController.placeholderText = "With Chips"
+    inputController = MDCTextInputControllerOutlined(textInput: textField)
+    inputController.placeholderText = "Regular MDCTextField"
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
```

Comment:
Tip: A cleaner Approach in swift to initialization (in general and of view controllers in particular) is to use self-executing closures to initialize properties instead of doing that in an init. (ie: lazy initializers for "chipInputController" and "inputController"). Then you will not need any initializers for the view controller.

Reply:
Seems there is a problem with the github diff. Going to rebase this branch and check the file diff there.